### ISSUE TITLE
Crash on launch with 1.16.5 version

### ISSUE THREAD

**Issue by redrapptor** [infernalstudios/Celestial-Configuration#2]
[Celestial Configuration Release 1.0.0 for Minecraft 1.16.5](https://www.curseforge.com/minecraft/mc-mods/sun-and-moon-celestial-configuration/files/3934445) causes my game to crash upon launch for some reason. With all of my mods enabled except for Celestial Config, it launches perfectly fine, but having it installed makes it impossible for me to reach the main menu without crashing.

I know very little about the inner workings of things, but the crash log seems to suggest that the mod is requiring Java Runtime  version 61.0 which apparently equates to Java 17; the version required by 1.18.

The 1.16.5 version of the mod claims to support Java 8, but I think something might've gotten messed up by accident.

Would a fix be possible or does it seem like it might be an issue on my end?

[celestial config crash.txt](https://paste.gg/5bae1e27f5c84af9bc0d3c4c1e0b957f)

**Comment by SwanX1** [comment]
Are you sure you've downloaded the correct file?
Please send the download page you got the mod from, as well as the mod file itself.
Double check if they're the same file, by downloading the mod file again and testing if the issue persists, and only then upload it to GitHub.

P.S. Sorry for the late response!

**Comment by SwanX1** [comment]
Hi again! There's no need for a follow-up from you, I've looked into it further and determined how to solve your issue!

It seems we have (more specifically, @LudoCrypt has) set the wrong Java version in our compilation configuration.

Problematic line: [`1.16.5/1.0.0`/build.gradle#L27](https://github.com/infernalstudios/Celestial-Configuration/blob/1.16.5/1.0.0/build.gradle#L27)

I've fixed this in a development environment and will push and publish the changes ASAP!
